controllers/artistController.js

```diff
@@ -0,0 +1,25 @@
+import Artist from '../services/Artist';
+
+export const getArtistById = async (req, res) => {
+  const { artist_id } = req.params;
+  const artist = await Artist.getArtistById(artist_id);
+  const { artist_plays } = await Artist.getArtistPlays(artist_id);
+  const songs = await Artist.getSongsByArtistId(artist_id);
+  const albums = await Artist.getAlbumsByArtistId(artist_id);
```

Comment:
wasn't easier to get all data of one artist in one call to DB by id, these are expensive calls, they should be reduced to minimum.